invesalius/gui/frame.py

```diff
@@ -1431,6 +1521,7 @@ def __init_items(self):
 
         # Add log viewer and error handling test menu items
         tools_menu.Append(ID_SHOW_LOG_VIEWER, _("Show Log Viewer"))
+        tools_menu.Append(ID_INTERACTIVE_SHELL, _("Interactive Shell"))
```

Comment:
I don't know but I think it's good idea to only enable this when development mode, maybe use some environment variable or some configuration. What do you think @paulojamorim and @rmatsuda ?

Reply:
Not sure which environment or configuration would be best to activate that. Maybe just a warning saying this feature is intended for development proposals, something along those lines.